crates/daphne-service-utils/src/durable_requests/bindings/aggregate_store.rs

```diff
@@ -128,6 +133,13 @@ impl DurableRequestPayload for AggregateStoreMergeReq {
                 };
             }
         }
+        {
+            let AggregateStoreMergeOptions {
```

Comment:
Is this destructuring required? Can we do `*options.skip_replay_protection`?

Reply:
Destructuring here guarantees that if this struct changes this code stops compiling, which is a good thing because we probably need to change it

Reply:
Nice! I like it